HW_3/register.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base.html" %}
+{% block content %}
+<h1>Registration</h1>
+<form method="POST" action="{{ url_for('register') }}">
+    {{ form.csrf_token }}
+    {% for field in form if field.name != 'csrf_token' %}
+        <p>
+            {{ field.label }}<br>
+            {{ field }}
+            {% if field.errors %}
+            <ul class="alert alert-danger">
+                {% for error in field.errors %}
+                <li>{{ error }}</li>
+                {% endfor %}
+            </ul>
+            {% endif %}
+        </p>
+    {% endfor %}
+    <p>
+        <input type="submit" value="Register">
+    </p>
```

Comment:
Тогда мне придется указывать эту кнопку.
```suggestion
    <p>
        {{ form.submit() }}
    </p>
```